community/command-line/src/main/java/org/neo4j/commandline/admin/AdminTool.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.commandline.admin;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import org.neo4j.helpers.Service;
+import org.neo4j.helpers.collection.Iterables;
+
+import static java.lang.String.format;
+
+public class AdminTool
+{
+    public static void main( String[] args )
+    {
+        Path homeDir = Paths.get( System.getenv( "NEO4J_HOME" ) );
+        Path configDir = Paths.get( System.getenv( "NEO4J_CONF" ) );
+        String extraHelp = System.getenv( "NEO4J_EXTRA_HELP" );
+        boolean debug = System.getenv( "NEO4J_DEBUG" ) != null;
+
+        new AdminTool( new ServiceCommandLocator(), System.out::println, extraHelp, debug )
+                .execute( homeDir, configDir, args )
+                .exit();
+    }
+
+    private final String scriptName = "neo4j-admin";
+    private final CommandLocator locator;
+    private final Output out;
+    private final boolean debug;
+    private final Usage usage;
+
+    public AdminTool( CommandLocator locator, Output out, String extraHelp, boolean debug )
```

Comment:
I'm trying to write a test for a command, and would like to verify that they are picked up correctly. It'd be helpful if there was a public overload of the `AdminTool` constructor that didn't take a CommandLocator argument, but used the `ServiceCommandLocator` as default.